librustzcash/Cargo.toml

```diff
@@ -24,6 +24,6 @@ libc = "0.2"
 pairing = { path = "../pairing" }
 lazy_static = "1"
 byteorder = "1"
-rand_core = "0.5"
+rand_core = "^0.5.1"
```

Comment:
Nit: we don't generally specify carets as they are the default behaviour.

Reply:
Ah, TIL.